docs/source/tutorial.rst

```diff
@@ -6,67 +6,55 @@ Tutorial
 Submit a dataset
 ****************
```

Comment:
Okay to not be super-clear yet, but this doesn't seem to mention that dimensions can now be directly included in the dataset.toml? (You don't necessarily need a dimensions.toml.) Possible to at least mention that so we'll remember to clean up and make more robust documentation later?

Reply:
I did actually refer to dataset.toml on line 17.

I do agree that this entire page needs enhancement.

Reply:
Maybe add TODOs that describe what kinds of updates need to be made? (Since I think it's higher priority right now just to get this in and get data to ReEDS?)